app/src/components/common/bar_diagram_probabilities/index.tsx

```diff
@@ -0,0 +1,108 @@
+import React, { HTMLAttributes } from 'react'
+import styled from 'styled-components'
+
+interface StyleProps {
+  isGreaterThanFifty?: boolean
+  barPercentage?: number
+  theme?: any
+}
+
+const OutcomeBar = styled.div`
+  align-items: center;
+  border-sizing: border-box;
+  border-collapse: collapse;
+  color: #00193c;
+  display: flex;
+  font-size: 13;
+  height: 38px;
+  line-height: 21px;
+  min-width: 200px;
+  text-size-adjust: 100%;
+`
+
+const Probability = styled.div<StyleProps>`
+  border-color: ${props =>
+    props.isGreaterThanFifty ? props.theme.colors.primarySoft : props.theme.colors.secondarySoft};
+  border-bottom-style: solid;
+  border-bottom-width: 2px;
+  display: block;
+  flex-basis: 0%;
+  flex-grow: 1;
+  flex-shrink: 1;
+  height: 34px;
+  margin-bottom: 2px;
+  margin-left: 0px;
+  margin-right: 0px;
+  margin-top: 2px;
+  position: relative;
+  text-align: center;
+`
+
+const Bar = styled.div<StyleProps>`
+  background-color: ${props =>
+    props.isGreaterThanFifty ? props.theme.colors.primarySoft : props.theme.colors.secondarySoft};
+  color: #00193c;
+  display: block;
+  height: 32px;
+  padding-bottom: 4px;
+  padding-left: 2px;
+  padding-right: 2px;
+  padding-top: 4px;
+  text-align: center;
+  width: ${props => props.barPercentage || '50'}%;
+`
+
+const LabelAmount = styled.div`
+  color: #526877;
+  display: block;
+  font-size: 13px;
+  font-weight: 400;
+  line-height: 0px;
+  position: absolute;
+  right: 10px;
+  text-align: center;
+  top: 16px;
+`
+
+const LabelOutcome = styled.div`
+  color: #00193c;
+  display: block;
+  left: 10px;
+  line-height: 0px;
+  position: absolute;
+  text-align: center;
+  top: 16px;
+  font-weight: 700;
+`
+
+const OutcomeName = styled.span`
+  color: #00193c;
+  line-height: 0px;
+  text-align: center;
+  text-size-adjust: 100%;
+  font-weight: 700;
+  font-size: 13px;
+`
+
+interface Props extends HTMLAttributes<HTMLButtonElement> {
+  outcomeName: string
+  probability: number
+}
+
+export const BarDiagram: React.FC<Props> = (props: Props) => {
+  const { outcomeName, probability } = props
+
+  const isGreaterThanFifty = probability > 50
+
+  return (
+    <OutcomeBar>
+      <Probability isGreaterThanFifty={isGreaterThanFifty}>
```

Comment:
Maybe we can make this a little more future proof and use something like `isWinning`? Of course, that should be something that receives from its parent.

Reply:
Sure, I update the variable name to `isWinning`, and is received from its parent